plugins/no_sir.py

```diff
@@ -0,0 +1,18 @@
+import re
+
+from errbot import BotPlugin, re_botcmd
+
+
+class no_sir(BotPlugin):
+  """
+  Do not use sir
+  """
+  def callback_message(self, msg):
+    emots = [':D']
+    if match_sir:
```

Comment:
@AkshJain99 What are you trying to do here? You instead should use a regex expression because the word `sir` could be present anywhere in the message.